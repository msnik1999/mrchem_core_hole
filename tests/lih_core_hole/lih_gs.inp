# vim:syntax=sh:

world_prec = 1.0e-3                 # Overall relative precision
world_size = 6                      # Size of simulation box 2^n

MPI {
  numerically_exact = true          # Guarantee identical results in MPI
}

Molecule {
$coords
Li  -3.787412196322  2.239543797522  0.016520727434
H  -6.044538550553  0.576167811227  1.170420150603
$end
}

WaveFunction {
  method = PBE                      # Wave function method (HF or DFT)
}

SCF {
  run = false
  localize = true                   # Use canonical of localized orbitals
  write_orbitals = true
  path_orbitals = initial_guess
}