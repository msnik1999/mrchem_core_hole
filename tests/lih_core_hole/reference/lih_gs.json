{
    "input": {
      "constants": {
        "N_a": 6.02214076e+23,
        "angstrom2bohrs": 1.8897261246257702,
        "boltzmann_constant": 1.380649e-23,
        "dipmom_au2debye": 2.5417464739297717,
        "e0": 8.8541878128e-12,
        "electron_g_factor": -2.00231930436256,
        "elementary_charge": 1.602176634e-19,
        "fine_structure_constant": 0.0072973525693,
        "hartree2ev": 27.211386245988,
        "hartree2kcalmol": 627.5094740630558,
        "hartree2kjmol": 2625.4996394798254,
        "hartree2simagnetizability": 78.9451185,
        "hartree2wavenumbers": 219474.6313632,
        "light_speed": 137.035999084,
        "meter2bohr": 18897261246.2577
      },
      "geom_opt": {
        "init_step_size": -0.5,
        "max_force_component": 0.005,
        "max_history_length": 10,
        "max_iter": 100,
        "minimal_step_size": 0.01,
        "run": false,
        "subspace_tolerance": 0.001,
        "use_previous_guess": false
      },
      "molecule": {
        "charge": 0,
        "coords": [
          {
            "atom": "li",
            "r_rms": 4.0992133976e-05,
            "xyz": [
              -3.787412196322,
              2.239543797522,
              0.016520727434
            ]
          },
          {
            "atom": "h",
            "r_rms": 2.6569547399e-05,
            "xyz": [
              -6.044538550553,
              0.576167811227,
              1.170420150603
            ]
          }
        ],
        "multiplicity": 1
      },
      "mpi": {
        "bank_size": -1,
        "numerically_exact": true,
        "omp_threads": -1,
        "shared_memory_size": 10000
      },
      "mra": {
        "basis_order": 5,
        "basis_type": "interpolating",
        "boxes": [
          2,
          2,
          2
        ],
        "corner": [
          -1,
          -1,
          -1
        ],
        "max_scale": 20,
        "min_scale": -5
      },
      "printer": {
        "file_name": "gs",
        "print_constants": false,
        "print_level": 0,
        "print_mpi": false,
        "print_prec": 6,
        "print_width": 75
      },
      "rsp_calculations": {},
      "scf_calculation": {
        "fock_operator": {
          "coulomb_operator": {
            "poisson_prec": 0.001,
            "shared_memory": false
          },
          "exchange_operator": {
            "exchange_prec": -1.0,
            "poisson_prec": 0.001
          },
          "kinetic_operator": {
            "derivative": "abgv_55"
          },
          "nuclear_operator": {
            "nuclear_model": "point_like",
            "proj_prec": 0.001,
            "shared_memory": false,
            "smooth_prec": 0.001
          },
          "xc_operator": {
            "shared_memory": false,
            "xc_functional": {
              "cutoff": 0.0,
              "functionals": [
                {
                  "coef": 1.0,
                  "name": "pbe"
                }
              ],
              "spin": false
            }
          }
        },
        "initial_guess": {
          "environment": "None",
          "external_field": "None",
          "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
          "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
          "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
          "file_basis": "initial_guess/mrchem.bas",
          "file_chk": "checkpoint/phi_scf",
          "file_gto_a": "initial_guess/mrchem.moa",
          "file_gto_b": "initial_guess/mrchem.mob",
          "file_gto_p": "initial_guess/mrchem.mop",
          "file_phi_a": "initial_guess/phi_a_scf",
          "file_phi_b": "initial_guess/phi_b_scf",
          "file_phi_p": "initial_guess/phi_p_scf",
          "localize": true,
          "method": "DFT (PBE)",
          "prec": 0.001,
          "relativity": "None",
          "restricted": true,
          "screen": 12.0,
          "type": "sad_gto",
          "zeta": 0
        },
        "occupancies": [],
        "properties": {
          "dipole_moment": {
            "dip-1": {
              "operator": "h_e_dip",
              "precision": 0.001,
              "r_O": [
                0.0,
                0.0,
                0.0
              ]
            }
          }
        },
        "write_orbitals": {
          "file_phi_a": "initial_guess/phi_a_scf",
          "file_phi_b": "initial_guess/phi_b_scf",
          "file_phi_p": "initial_guess/phi_p_scf"
        }
      },
      "schema_name": "mrchem_input",
      "schema_version": 1
    },
    "output": {
      "properties": {
        "center_of_mass": [
          -4.073619909235116,
          2.0286247074765114,
          0.1628372658138295
        ],
        "charge": 0,
        "dipole_moment": {
          "dip-1": {
            "magnitude": 2.620016018850234,
            "r_O": [
              0.0,
              0.0,
              0.0
            ],
            "vector": [
              1.9505411837820255,
              1.4372584568889355,
              -0.9970763047546964
            ],
            "vector_el": [
              19.357316323299735,
              -5.857540746903565,
              -2.217058637659732
            ],
            "vector_nuc": [
              -17.40677513951771,
              7.2947992037925005,
              1.2199823329050354
            ]
          }
        },
        "geometry": [
          {
            "symbol": "Li",
            "xyz": [
              -3.787412196322,
              2.239543797522,
              0.016520727434
            ]
          },
          {
            "symbol": "H",
            "xyz": [
              -6.044538550553,
              0.576167811227,
              1.170420150603
            ]
          }
        ],
        "multiplicity": 1,
        "orbital_energies": {
          "energy": [
            -1.8535835335291997,
            -0.07087089647732694
          ],
          "occupation": [
            2.0,
            2.0
          ],
          "spin": [
            "p",
            "p"
          ],
          "sum_occupied": -3.8489088600130534
        },
        "orbital_positions": [
          [
            -3.7857881859019864,
            2.2407384951324114,
            0.015694028416145388
          ],
          [
            -5.892869975747877,
            0.6880318783193707,
            1.0928352904137204
          ]
        ],
        "scf_energy": {
          "E_ee": 5.7373552094219935,
          "E_eext": 0.0,
          "E_el": -8.952883480791225,
          "E_en": -20.444461308928275,
          "E_kin": 8.013021843048907,
          "E_next": 0.0,
          "E_nn": 0.9894513494461199,
          "E_nuc": 0.9894513494461199,
          "E_tot": -7.9634321313451055,
          "E_x": 0.0,
          "E_xc": -2.2587992243338513,
          "Er_el": 0.0,
          "Er_nuc": 0.0,
          "Er_tot": 0.0
        }
      },
      "provenance": {
        "creator": "MRChem",
        "mpi_processes": 1,
        "nthreads": 8,
        "routine": "mrchem.x",
        "total_cores": 8,
        "version": "1.2.0-alpha"
      },
      "rsp_calculations": null,
      "scf_calculation": {
        "initial_energy": {
          "E_ee": 5.7373552094219935,
          "E_eext": 0.0,
          "E_el": -8.952883480791225,
          "E_en": -20.444461308928275,
          "E_kin": 8.013021843048907,
          "E_next": 0.0,
          "E_nn": 0.9894513494461199,
          "E_nuc": 0.9894513494461199,
          "E_tot": -7.9634321313451055,
          "E_x": 0.0,
          "E_xc": -2.2587992243338513,
          "Er_el": 0.0,
          "Er_nuc": 0.0,
          "Er_tot": 0.0
        },
        "success": true
      },
      "schema_name": "mrchem_output",
      "schema_version": 1,
      "success": true
    }
  }