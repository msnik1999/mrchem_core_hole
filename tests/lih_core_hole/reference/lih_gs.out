***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         mom_merged                               ***
***         Git commit hash    a1a2e5991d8adbd8494b-dirty               ***
***         Git commit author  Niklas GÃ¶llmann                         ***
***         Git commit date    Fri May 10 13:22:30 2024 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               8
 Total cores             :                                               8

---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.6.0-alpha
 Git branch            : mom
 Git commit hash       : 9ebc1da6290a65a252e6
 Git commit author     : Laura Ratcliff
 Git commit date       : Wed Jun 14 14:11:39 2023 +0200

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (8 threads)

---------------------------------------------------------------------------



===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================



***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************


===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :       -3.787412        2.239544        0.016521
    1       H            :       -6.044539        0.576168        1.170420
---------------------------------------------------------------------------
 Center of mass          :       -4.073620        2.028625        0.162837
===========================================================================



***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                           2.000
 Beta electrons          :                                           2.000
 Total electrons         :                                           4.000
---------------------------------------------------------------------------
    n     Occ Spin       :                                            Norm
---------------------------------------------------------------------------
    0    2.00    p       :                              9.999994078667e-01
    1    2.00    p       :                              9.999982305380e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE)
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  8.013021843049
 E-N energy              :            (au)                -20.444461308928
 Coulomb energy          :            (au)                  5.737355209422
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -2.258799224334
 N-N energy              :            (au)                  0.989451349446
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -8.952883480791
 Nuclear energy          :            (au)                  0.989451349446
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.963432131345e+00
                         :      (kcal/mol)             -4.997129108477e+03
                         :        (kJ/mol)             -2.090798818987e+04
                         :            (eV)             -2.166960275697e+02
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n     Occ Spin       :                                         Epsilon
---------------------------------------------------------------------------
    0    2.00    p       :            (au)                 -1.853583533529
    1    2.00    p       :            (au)                 -0.070870896477
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -3.848908860013
===========================================================================



***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************


===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :       -3.787412        2.239544        0.016521
    1       H            :       -6.044539        0.576168        1.170420
---------------------------------------------------------------------------
 Center of mass          :       -4.073620        2.028625        0.162837
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  8.013021843049
 E-N energy              :            (au)                -20.444461308928
 Coulomb energy          :            (au)                  5.737355209422
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -2.258799224334
 N-N energy              :            (au)                  0.989451349446
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -8.952883480791
 Nuclear energy          :            (au)                  0.989451349446
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.963432131345e+00
                         :      (kcal/mol)             -4.997129108477e+03
                         :        (kJ/mol)             -2.090798818987e+04
                         :            (eV)             -2.166960275697e+02
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n     Occ Spin       :                                         Epsilon
---------------------------------------------------------------------------
    0    2.00    p       :            (au)                 -1.853583533529
    1    2.00    p       :            (au)                 -0.070870896477
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -3.848908860013
===========================================================================


===========================================================================
                             Orbital Positions
---------------------------------------------------------------------------
Index                    :               x               y               z
---------------------------------------------------------------------------
    0                    :       -3.785788        2.240738        0.015694
    1                    :       -5.892870        0.688032        1.092835
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       19.357316       -5.857541       -2.217059
 Magnitude               :            (au)                       20.345315
                         :         (Debye)                       51.712632
---------------------------------------------------------------------------
 Nuclear vector          :      -17.406775        7.294799        1.219982
 Magnitude               :            (au)                       18.912913
                         :         (Debye)                       48.071830
---------------------------------------------------------------------------
 Total vector            :        1.950541        1.437258       -0.997076
 Magnitude               :            (au)                        2.620016
                         :         (Debye)                        6.659416
===========================================================================




***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  8s                       ***
***                                                                     ***
***************************************************************************